react-native-libraries.json

```diff
@@ -15284,5 +15284,15 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+    {
+    "githubUrl": "https://github.com/jpudysz/react-native-turbo-mock-location-detector",
+    "npmPkg": "react-native-turbo-mock-location-detector",
```

Comment:
```suggestion
  {
    "githubUrl": "https://github.com/jpudysz/react-native-turbo-mock-location-detector",
```

Nits.

Reply:
hi @Simek, just to know for future PRs, what purpose npmPkg serves? Thanks

Reply:
It's mostly for scope packages, or ones which have the code inside monorepos. We explain it in the README file:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg